docs/static/releasenotes.asciidoc

```diff
@@ -12,6 +13,75 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-2-0]]
+=== Logstash 8.2.0 Release Notes
+
+60af28a0f - update google java format dependency (#13700) https://github.com/elastic/logstash/pull/13700
+43c657634 - Improve safety of stats retrieval from queue (#13942) https://github.com/elastic/logstash/pull/13942
+* Update failing policy in Central Management fetcher and license checker if hit ES down node https://github.com/elastic/logstash/pull/13689[#13689]
+* Print bundled jdk's version in launch scripts when `LS_JAVA_HOME` is provided https://github.com/elastic/logstash/pull/13880[#13880]
+* Update Jackson databind in ingest-converter tool https://github.com/elastic/logstash/pull/13900[#13900]
+* Fix/flaky error tests in missed retries to central management https://github.com/elastic/logstash/pull/13941[#13941]
+d2739a875 - Updates the LogstashService launcher to use the JDK specified (#13946) (4 weeks ago) <Andrea Selva>
+
+
+* [DOC] precedence between environment variables and yml in docker https://github.com/elastic/logstash/pull/12198[#12198]
+* [doc] Remove references to sysv and upstart scripts when running logstash as a service. https://github.com/elastic/logstash/pull/13953[#13953]
+* Backport PR #13922 to 8.2: Removal of sys-v init.d scripts https://github.com/elastic/logstash/pull/13954[#13954]
+* Backport PR #13953 to 8.2: [doc] Remove references to sysv and upstart scripts when running logstash as a service. https://github.com/elastic/logstash/pull/13955[#13955]
+* Backport PR #13981 to 8.2: Fix window CI for PQ size checking https://github.com/elastic/logstash/pull/13987[#13987]
+* Backport PR #13973 to 8.2: [doc] Add Logstash to Logstash HTTP example configuration. https://github.com/elastic/logstash/pull/13999[#13999]
+* Backport PR #13834 to 8.2: Doc: Refine content for generated CA in LS-ES security section https://github.com/elastic/logstash/pull/14012[#14012]
+* Backport PR #13990 to 8.2: Doc: Restructure source files for pipeline configuration https://github.com/elastic/logstash/pull/14015[#14015]
+* Backport PR #14021 to 8.2: Fix Bundled JDK docs to reflect JDK 11 https://github.com/elastic/logstash/pull/14022[#14022]
+
+
+
+==== Plugins
+
+*Http Filter - 1.4.0*
+
+* Feat: added ssl_supported_protocols option https://github.com/logstash-plugins/logstash-filter-http/pull/38[#38]
+
+*Kv Filter - 4.7.0*
+
+* Allow attaching multiple tags on failure. The `tag_on_failure` option now also supports an array of strings https://github.com/logstash-plugins/logstash-filter-kv/issues/92[#92]
+
+*Beats Input - 6.3.0*
+
+* Added support for TLSv1.3. https://github.com/logstash-plugins/logstash-input-beats/pull/447[#447]
+
+*Elasticsearch Input - 4.12.3*
+
+* Fix: update Elasticsearch Ruby client to correctly customize 'user-agent' headerhttps://github.com/logstash-plugins/logstash-input-elasticsearch/pull/171[#171]
```

Comment:
```suggestion
* Fix: update Elasticsearch Ruby client to correctly customize 'user-agent' header https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/171[#171]
```

Reply:
Space is required for link to render properly. (I've already fixed this in the changelog.)